src/shared/infrastructure/profile-metrics/profile-metrics.json

```diff
@@ -113,28 +113,28 @@
   },
   "process_cpu:lock_count:count:cpu:nanoseconds": {
     "id": "process_cpu:lock_count:count:cpu:nanoseconds",
-    "description": "Count of lock acquisitions attempted during CPU time",
+    "description": "Number of lock acquisitions attempted during CPU time",
     "type": "lock_count",
     "group": "locks",
     "unit": "short"
   },
   "process_cpu:lock_time:nanoseconds:cpu:nanoseconds": {
     "id": "process_cpu:lock_time:nanoseconds:cpu:nanoseconds",
-    "description": "Cumulative time spent acquiring locks, in nanoseconds",
+    "description": "Cumulative time spent acquiring locks",
```

Comment:
Again, do we want to specify nanoseconds to help people understand what they are looking at? Or do you think it's immediately obvious?

Reply:
I find specifying units is tricky, and I would consistently remove them (bytes, nanoseconds).

 This is why:

The headline says it is nanoseconds, but the graph's axis is actually say the correct seconds:

![image](https://github.com/user-attachments/assets/2e42c009-cedf-4b61-83e0-9a9795922b5a)